hawkular-alerts-bus/src/main/java/org/hawkular/alerts/bus/listener/AvailDataListener.java

```diff
@@ -115,7 +115,7 @@ protected void onBasicMessage(AvailDataMessage msg) {
     protected String convertReceivedMessageClassNameToDesiredMessageClassName(String className) {
 
         if (className.equals("org.hawkular.metrics.component.publish.AvailDataMessage")) {
-            return "org.hawkular.alerts.bus.messages.AvailDataMessage";
+            return "org.hawkular.alerts.bus.api.AvailDataMessage";
```

Comment:
Your suggestion is also correct.
I just followed the original code as I think is more readable and this class shouldn't change.
I will let to Jay to decide it.
Also, I've created a JIRA to improve the itest for this scenario to check in the next versions that things are working properly.
https://issues.jboss.org/browse/HWKALERTS-121